lute/cli/commands/transform/init.luau

```diff
@@ -1,28 +1,13 @@
 local fs = require("@lute/fs")
-local luau = require("@lute/luau")
+local luau = require("@std/luau")
 local pathLib = require("@std/path")
 
 local arguments = require("@self/lib/arguments")
 local files = require("@self/lib/files")
 local types = require("@self/lib/types")
 
-local function exhaustiveMatch(value: never): never
```

Comment:
Was removing this function intentional? I still see it referenced later in the file

Reply:
no definitely a mistake, i'll add it back rn